client/app/scripts/liveblog-edit/timeline.js

```diff
@@ -197,12 +197,42 @@ define([
             }
         };
     }])
-    .directive('itemActions', ['api', 'notify', 'gettext', 'asset', 'itemsService', 'modal',
+    .directive('postActions', ['api', 'notify', 'gettext', 'asset', 'itemsService', 'modal',
         function(api, notify, gettext, asset, itemsService, modal) {
         return {
             restrict: 'E',
             replace: true,
-            templateUrl: 'scripts/liveblog-edit/views/timeline-item-actions.html',
+            templateUrl: 'scripts/liveblog-edit/views/timeline-post-actions.html',
+            link: function(scope, elem, attrs) {
+                scope.removePost = function() {
+                    var remItems = scope.post.groups[1].refs;
+                    confirm().then(function() {
+                        api.posts.remove(scope.post).then(function(message) {
+                            notify.pop();
+                            notify.info(gettext('Post removed'));
+                            scope.remove({post:scope.post});
+                            for (var i = 0, item; i < remItems.length; i ++) {
```

Comment:
@vied12 I don't agree with this, even if I don't like so much the `business logic` inside the client, if this will complicate a lot the server or even make more unreadable the server code we need to make it on the client.
also remember that is no transaction  on the server alswell so it can `fail` aswell on the same steps.